{{cookiecutter.app_name}}/{% if cookiecutter.use_react == "yes" %}webpack.config.js{% endif %}

```diff
@@ -37,7 +37,7 @@ module.exports = {
     new HtmlWebpackPlugin({
       template: './{{cookiecutter.app_name}}/static/react/index.html'
     }),
-    new CleanWebpackPlugin(['{{cookiecutter.app_name}}/static/react/dist/'])
+    new CleanWebpackPlugin()
```

Comment:
This has been updated on CleanWebpackPlugin. No need for this because it defaults to output path: https://github.com/johnagan/clean-webpack-plugin/issues/103